src/content/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards.mdx

```diff
@@ -107,6 +107,22 @@ Here you can carry out the following actions:
     To organize dashboards with multiple pages, see [Add pages to a dashboard](/docs/dashboards/new-relic-one-dashboards/manage-your-dashboard/add-pages-dashboards).
   </Collapser>
 
+  <Collapser
+    id="dashboards-import"
+    title="Import a dashboard"
+  >
+    You can import a dashboard as a JSON by selecting the **Import a dashboard** button located at the top-right corner of the dashboards index:
```

Comment:
Here, I'd remove "a" and make it instead, "import a dashboard as JSON by selecting.."  Or, "import a dashboard as JSON code by selecting..."

If you leave the article, add a noun, like file: import a dashboard as a JSON file... "